### PR TITLE
Add a new item to the known issue list

### PR DESCRIPTION
**Pull request by yiheng**
## What changes were proposed in this pull request?
Add a new item to the known issue list

## How was this patch tested?
doc change, manul reviewed

## Related links or issues (optional)
fixed https://github.com/intel-analytics/BigDL/issues/2009

### CODE REVIEW

File: docs/docs/known-issues.md [hunk 1]
```diff
@@ -1,18 +1,34 @@
...
+expected to be a multiple of **total cores** used in the job.
 
-* You may observe very poor performance when running BigDL for Spark 2.0 with Java 7; it is highly recommended to use Java 8 when building and running BigDL for Spark 2.0.
+* You may observe very poor performance when running BigDL for Spark 2.0 with Java 7; it is highly
+recommended to use Java 8 when building and running BigDL for Spark 2.0.
 
-* On Spark 2.0, please use default Java serializer instead of Kryo because of [Kryo Issue 341](https://github.com/EsotericSoftware/kryo/issues/341). The issue has been fixed in Kryo 4.0. However, Spark 2.0 uses Kryo 3.0.3. Spark 1.5 and 1.6 do not have this problem.
+* On Spark 2.0, please use default Java serializer instead of Kryo because of
+[Kryo Issue 341](https://github.com/EsotericSoftware/kryo/issues/341). The issue has been fixed in
+Kryo 4.0. However, Spark 2.0 uses Kryo 3.0.3. Spark 1.5 and 1.6 do not have this problem.
```
Comment (jason-dai) [inline docs/docs/known-issues.md]: Do you know if Spark 2.1 or 2.2 still have this problem?

File: docs/docs/known-issues.md [hunk 2]
```diff
@@ -1,18 +1,34 @@
...
-* On Windows, if you meet "Could not locate executable null\bin\winutils.exe" error, you need to install winutils.exe. Please refer this [post](https://stackoverflow.com/questions/35652665/java-io-ioexception-could-not-locate-executable-null-bin-winutils-exe-in-the-ha).
+* On Windows, if you meet "Could not locate executable null\bin\winutils.exe" error, you need to
+install winutils.exe. Please refer this
+[post](https://stackoverflow.com/questions/35652665/java-io-ioexception-could-not-locate-executable-null-bin-winutils-exe-in-the-ha).
+
+* If the training data are cached before all the executor resources are allocated(this sometimes
+happens when the data set is too small), we find Spark may not distribute the training data
+partitions evenly on each executor. So the training tasks will be unbalanced allocated among
+executors. To solve this problem, you can increase the
+`spark.scheduler.maxRegisteredResourcesWaitingTime` property(default is 30s).
```
Comment (jason-dai) [inline docs/docs/known-issues.md]: Shall we just set a default value for it in BigDL?

File: docs/docs/known-issues.md (hunk 1 above)
Comment (yiheng) [reply docs/docs/known-issues.md]: just check the master code. it still uses kryo 3.0.3

File: docs/docs/known-issues.md (hunk 2 above)
Comment (yiheng) [reply docs/docs/known-issues.md]: which value is appropriate, is 60s okay? Even set it larger I think we still keep this in the known issue list

Comment (jason-dai) [reply docs/docs/known-issues.md]: Why 30s be an issues? It's long enough.

Comment (yiheng) [reply docs/docs/known-issues.md]: It is related to user's environment...

**Pull request merged**